ambry-api/src/main/java/com/github/ambry/account/Container.java

```diff
@@ -353,6 +371,7 @@ private Container(JSONObject metadata, short parentAccountId) throws JSONExcepti
         id = (short) metadata.getInt(CONTAINER_ID_KEY);
         name = metadata.getString(CONTAINER_NAME_KEY);
         status = ContainerStatus.valueOf(metadata.getString(STATUS_KEY));
+        lastUpdateTime = metadata.getLong(CONTAINER_LAST_UPDATE_TIME_KEY);
```

Comment:
any new field added to an existing version should be optional so existing records can be deserialized by the new code.

Reply:
In version1, I assigned the default value which is 0. In version2, I made it optional for the new field.